docs/guide/controller-programming.md

```diff
@@ -18,10 +20,67 @@ int main() {
     printf("Hello World!\n");
 
   wb_robot_cleanup();
+  return 0;
+}
+```
+%tab-end
+
+%tab "C++"
+```cpp
+#include <iostream>
+#include <webots/Robot.hpp>
+
+using namespace webots;
+
+int main() {
+  Robot *robot = new Robot();
 
+  while (robot->step(32) != -1)
```

Comment:
Should we use a define for here ? We are talking about those specificities later but the next example is using it nonetheless...

Reply:
I would not use a define or variable here as putting the hard-coded value makes it simpler to read (this is a tutorial).